src/cui.cpp

```diff
@@ -334,29 +335,46 @@ void show_ncurses(Line *lines[], int nproc) {
   if (cols > PROGNAME_WIDTH)
     cols = PROGNAME_WIDTH;
 
-  proglen = cols - 65;
+
+ //issue #110 - dynamic length devicename
+ int devlen = MIN_COLUMN_WIDTH_DEV; int curlen;
```

Comment:
Looks neat! Perhaps move the logic to a private helper function that returns `devlen`?